### PR TITLE
Update ParallelWaveGAN config + Tacotron2 masked loss

### PR DESCRIPTION
**Pull request by iamanigeeit**
Referencing https://github.com/coqui-ai/TTS/issues/1187 and https://github.com/coqui-ai/TTS/issues/1192

I did not realize that a PR must include all commits in a branch and i should have split into a separate branch for the ParallelWaveGAN config and the Tacotron2 masked loss.

### CODE REVIEW

File: TTS/tts/configs/tacotron_config.py [hunk 1]
```diff
@@ -135,6 +134,8 @@ class TacotronConfig(BaseTTSConfig):
         ga_alpha (float):
             Weight for the guided attention loss. If set less than or equal to zero, it disables the corresponding loss
             function. Defaults to 5.
+        stopnet_alpha (float):
+            Weight for the guided attention loss. Defaults to 100.0
```
Comment (erogol) [inline TTS/tts/configs/tacotron_config.py:138]: This comment incorrectly explains the parameter.

File: TTS/vocoder/configs/parallel_wavegan_config.py [hunk 2]
```diff
@@ -48,31 +48,31 @@ class ParallelWaveganConfig(BaseGANVocoderConfig):
...
+            Generator model initial learning rate. Defaults to 0.0001.
         lr_disc (float):
-            Discriminator model initial learning rate. Defaults to 0.0002.
+            Discriminator model initial learning rate. Defaults to 0.00005.
         optimizer (torch.optim.Optimizer):
             Optimizer used for the training. Defaults to `AdamW`.
         optimizer_params (dict):
             Optimizer kwargs. Defaults to `{"betas": [0.8, 0.99], "weight_decay": 0.0}`
         lr_scheduler_gen (torch.optim.Scheduler):
-            Learning rate scheduler for the generator. Defaults to `ExponentialLR`.
```
Comment (erogol) [inline TTS/vocoder/configs/parallel_wavegan_config.py:71]: What is the reason for these config changes?

**Comment by erogol** [comment]
Thanks for the PR. As you said, it'd be better to have separate PRs.
